TypeScript.py

```diff
@@ -1,19 +1,13 @@
 import sys
+import os
 
-# Sublime/Python 2 & 3 differ in the name of this module, thus package import
-# needs to be handled slightly differently
-if sys.version_info < (3, 0):
-    from libs import *
-    from libs.reference import *
-    from libs.view_helpers import *
-    from listeners import *
-    from commands import *
-else:
-    from .libs import *
-    from .libs.reference import *
-    from .libs.view_helpers import *
-    from .listeners import *
-    from .commands import *
+sys.path.append(os.path.dirname(os.path.abspath(__file__)))
```

Comment:
One of the goals of the refactor was to remove the need to modify the global module resolution path.  This is a bad practice, and we should avoid doing this, (and don't need to).  

We should only special case the top level package import (as you had in the previous refactor), and have everything else be intra-package.  (I use a similar pattern in a prototype at https://github.com/billti/BillsSTP/blob/master/init.py ).